_includes/footer.html

```diff
@@ -0,0 +1,16 @@
+<footer class="footer">
+  <div class="container">
+    <div class="content has-text-centered">
+      <p>
+        Copyright © 2018 - RubyNepal - Powered by <a href="https://jekyllrb.com/" class="rnw-link">Jekyll</a>
```

Comment:
We could make this rolling year. What do you think? @zoras @RohitRox dai

Reply:
👍

Reply:
like this ```Copyright © {{ site.time | date: '%Y' }} - RubyNepal - Powered by``` ?

Reply:
yes